### ISSUE TITLE
Using the "All displays - Show screensaver on all connected displays" does NOT show the screensaver on ALL displays...

### ISSUE THREAD

**Issue by SYBR-biz** [opus1269/photo-screen-saver#10]
Using the "All displays - Show screensaver on all connected displays" does NOT show the screensaver on ALL displays... Only on one or another of the primary Chromebook display OR the attached HDMI display.

**Comment by opus1269** [comment]
Can you check what version of the extension you are running? You can see it from the main Chrome menu: More Tools -> Extensions. I thought I had fixed this in version 1.4.4.

**Comment by SYBR-biz** [comment]
1.7.1 (By the way, I just posted this issue again through the Chrome store, sorry. I forgot that I had posted here so long ago. Anyway, I am still having this issue.) I _love_ the extension! Just wish I could get this feature to work too.

**Comment by SYBR-biz** [comment]
FYI, I spent a little time fooling around turning other extensions off and on, and I have a hunch that this extension may be the culprit: New Tab, New Window: https://chrome.google.com/webstore/detail/new-tab-new-window/dndlcbaomdoggooaficldplkcmkfpgff?utm_source=chrome-app-launcher-info-dialog

**Comment by SYBR-biz** [comment]
And, if I use this extension (One Window*), then any monitor without a fullscreen (F11) window running, forces the image into a tabbed window. Interesting...? 
- https://chrome.google.com/webstore/detail/one-window/papnlnnbddhckngcblfljaelgceffobn?utm_source=chrome-app-launcher-info-dialog

**Comment by SYBR-biz** [comment]
Yeah, well, thought I had this solved, or diagnosed for conflicts with other extensions... But I was wrong. When using "All Displays" only shows in on the Chromebook display. When All Displays is off, only displays on the attached HDMI monitor. Disappointing.
